tutorials/math/bit-manipulation.md

```diff
@@ -226,7 +242,9 @@ $<<$ shifts the bits to the left. For example, $1 << 1 = 2$ because we shift the
 
 Similarily, $1 << 2 = 4$ because we shift the $1$ ($0001_2$) to the left twice to become $4$ ($0100_2$).
 
-And you may find that $1 << n$ is actually $2 ^ n$.
+And you may find that $1 << n$ is actually $2 ^ n$. Also $n << m$ means multiplying n by 2 power m. i.e, $n = n * (2^m)$. 
+
+In simple, $n << m$ _shifting each bit of n to left m times_. Let's say, $n = 8$ and $m = 2$. $n$ can be represented as $1000_2$ in binary, Therefore, $8 << 2$ = $1000_2$ << $2$ = $100000_2$ (32), which is same as $(8 * 2^2)$ = $32$
```

Comment:
`$8$ << $2$ = $1000_2$ << $2$ = $100000_2$ (32)`

Reply:
I'm not sure, what is the question here?

If the expression isn't clear with the concept, If so, Since the given example, was just only handled left shift (<<) operator with 1. 

What if we decide to right shift more than once, 2 times or 3 times with other than 1, this is to calculate the number with the given formula.

Reply:
Just formatting only. The context is same as yours.

Reply:
Cool.